docs/spec.yml

```diff
@@ -360,7 +473,100 @@ components:
         - scope
         - root_token_enabled
         - csrf_enabled
-  
+    Job:
+      type: object
+      properties:
+        id:
+          type: string
+        job_name:
+          type: string
+          description: Job Name
+        jenkins_server:
+          type: string
+          description: Jenkins Server
+        is_tag:
+          type: boolean
+          description: 'Ref type: true - tag, false - branch'
+        is_pipeline:
+          type: boolean
+          description: Multibranch Pipeline
+        triggers:
+          type: array
+          description: Triggers
+          items:
+            type: string
+            enum:
+              - 'add'
+              - 'push'
+              - 'manual'
+              - 'delete'
+              - 'prautomerged'
+              - 'propened'
+              - 'prreopened'
+              - 'prsourcerescoped'
+              - 'prdestrescoped'
+              - 'prmerged'
+              - 'prdeclined'
+              - 'prdeleted'
+              - 'prapproved'
+          uniqueItems: true
+          minItems: 1
+        token:
+          type: string
+          description: Trigger builds remotely (e.g., from scripts) or leave blank to use user API token
+        build_parameters:
+          type: string
```

Comment:
While the build_parameters are just a single multiline string in th UI, I don't think that makes a lot of sense intuitively (and I have been considering changing it).

Instead I think we'd want an array of objects with name and value fields like:

```
build_parameters:
    type: array
    items:
        type: object
        properties:
            name:
                type: string
            value:
                type: string
```

Reply:
It makes sense. Just one notice, in java code I found, that plugin can handle parameter without a value. I don't found a way, how to specify it in openapi spec (add default, or mark as not required field).
Addressed.